packages/core/src/components/GlobalActions/GlobalActions.tsx

```diff
@@ -60,62 +50,56 @@ export const HvGlobalActions = (props: HvGlobalActionsProps) => {
     position: positionProp,
     ...others
   } = useDefaultProps("HvGlobalActions", props);
-
-  const { activeTheme } = useHvTheme();
-
   const muiTheme = useTheme();
+  const { classes, cx, css } = useClasses(classesProp);
+  const isSmDown = useMediaQuery(muiTheme.breakpoints.down("sm"));
+  const isUpMd = useMediaQuery(muiTheme.breakpoints.up("md"));
+
+  const fixedPositionCss =
+    positionProp === "fixed" && isUpMd
+      ? {
+          width: `calc(100% - 2 * ${theme.spacing(4)})`,
+          marginLeft: `${theme.spacing(4)}`,
+          marginRight: `${theme.spacing(4)}`,
+        }
+      : isSmDown
+      ? {
+          width: `calc(100% - 2 * ${theme.spacing(2)})`,
+          marginLeft: `${theme.spacing(2)}`,
+          marginRight: `${theme.spacing(2)}`,
+        }
```

Comment:
This `isSmDown` styling should only be used if `positionProp` is fixed but in this case it's used even if it's not fixed.